README.md

```diff
@@ -4,10 +4,21 @@ This repository contains several sample applications that demonstrate various Ok
 
 Each sample makes use of the [Okta Angular Library][].
 
-Before running the samples, you will need to gather the following information from the Okta Developer Console:
+If you haven't done so already, register for a free account at [developer.okta.com](https://developer.okta.com/). Select **Create Free Account** and fill in the forms to complete the registration process. Once you are done and logged in, you will see your Okta Developer Console. 
 
-- **Client Id** - The client ID of the SPA application that you created earlier. This can be found on the "General" tab of an application, or the list of applications.  This identifies the application that tokens will be minted for.
-- **Issuer** - This is the URL of the authorization server that will perform authentication.  All Developer Accounts have a "default" authorization server.  The issuer is a combination of your Org URL (found in the upper right of the console home page) and `/oauth2/default`. For example, `https://dev-1234.oktapreview.com/oauth2/default`.
+> **Tip**: You can also create an account using the [Okta CLI](https://github.com/oktadeveloper/okta-cli) and `okta register`. To create an app, run `okta apps create` and use the settings below.
+
+Register your application by selecting **Applications** > **Add Application**. On the next screen, choose **Single Page App** and click **Next**.
+
+On the following screen, edit the application settings. For Angular applications running in developer mode, the port number should be 4200. Configure your app as follows:
+
+* **Base URI**: `http://localhost:4200`
+* **Login redirect URI**: `http://localhost:4200/callback` 
```

Comment:
I don't really recommend '/callback'.  This is not the default setting and assumes their app would have only one callback.

As deceptive as it is, I'd continue to use `/implicit/callback` until we change the default redirect_uri.  (at which point I'm advocating for `/login/callback`, though others have suggested `/oidc/callback`)

Is `4200` an angular convention?

Reply:
We've been using `/callback` in all our 2020 blog posts. @aaronpk recommended it. Why does using it prevent you from having `/secondcallback`? 

Yes, 4200 is an Angular convention. React uses 3000 and Vue uses 8080.

Reply:
I like the concept of meaningful urls.  If I have a callback, and /callback, I'll assume they are the same, but if I have 2 callbacks, I have no idea which one is /callback.

Regardless, it doesn't match our defaults, so I'm questioning if we want to require the extra step and easy point of confusion.

Reply:
Another point is that we've had problems with devs accidentally blocking access to routes, if we know they have `/login` accessible without authentication then `/login/callback` is likely free of complication.